lib/vsc/administration/vo.py

```diff
@@ -298,15 +319,26 @@ def _set_quota(self, storage_name, path, quota, fileset_name=None):
         """
         if not fileset_name:
             fileset_name = self.vo.vsc_id
+
         try:
-            # expressed in bytes, retrieved in KiB from the backend
-            hard = quota * 1024 * self.storage[self.host_institute][storage_name].data_replication_factor
-            soft = int(hard * self.vsc.quota_soft_fraction)
+            storage = self.storage[self.host_institute][storage_name]
+        except KeyError:
+            logging.exception("Trying to access non-existent storage: %s", storage_name)
+        else:
+            fs_backend = load_storage_operator(storage)
+
+        # expressed in bytes, retrieved in KiB from the account backend
+        hard = quota * 1024
+        if storage.backend == 'gpfs':
```

Comment:
why only for gpfs? 

and can this code be factored out?

Reply:
good point, since the replication factor defaults to 1 I changed it to always be applied and moved all this into a common method in `vsc.administration.tools`. See https://github.com/hpcugent/vsc-administration/pull/134/commits/06a978bc1b80d23e4c42306cd40816f7b60a020a